centreon-ha/SECURITY.md

```diff
@@ -0,0 +1,166 @@
+# Security Policy
```

Comment:
Could you delete this file ? If not, we may delete it in another PR

Reply:
done